support-frontend/assets/pages/contributions-landing/components/ContributionThankYou/ContributionThankYou.jsx

```diff
@@ -177,6 +180,11 @@ function ContributionThankYou(props: PropTypes) {
     return null;
   };
 
+  const store = pageInit(() => initReducer(), true);
```

Comment:
in general we should not access the store directly.
Inside a component, we use `mapStateToProps` to wire up the props to the parts of the store that we need.

Reply:
See https://react-redux.js.org/using-react-redux/connect-mapstate
And actually, this component is already receiving `countryId` from the store.

Reply:
Thanks for the feedback @tomrf1, v useful to know! I've made those changes now ✅